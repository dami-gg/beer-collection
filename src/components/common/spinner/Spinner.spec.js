import React from "react";
import Spinner from "./Spinner";

import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("Spinner", () => {
  let component;

  it("should render without crashing", () => {
    shallow(<Spinner />);
  });

  it("should match the snapshot", () => {
    component = renderer.create(<Spinner />);
    let json = component.toJSON();

    expect(json).toMatchSnapshot();
  });
});
