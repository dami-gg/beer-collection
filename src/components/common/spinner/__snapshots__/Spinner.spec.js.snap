exports[`Spinner should match the snapshot 1`] = `
<div
  className="spinner" />
`;
